File: HHTXT/Project.vb
Public Class Project

    Private Sub Project_FormClosed(ByVal sender As Object, ByVal e As System.Windows.Forms.FormClosedEventArgs) Handles Me.FormClosed
        MDIParent1.AmountOfProjects -= 1
    End Sub

    Private Sub Project_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    End Sub
End Class

File: HHTXT/Project.cs
using System;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class Project
    {
        public Project()
        {
            InitializeComponent();
        }

        private void Project_FormClosed(object sender, FormClosedEventArgs e)
        {
            MDIParent1.instance.AmountOfProjects -= 1;
        }

        private void Project_Load(object sender, EventArgs e)
        {

        }

        private void RichTextBox1_TextChanged(object sender, EventArgs e)
        {
            MacroManager.Instance.InvokeEvent(MacroTrigger.CONTENT_CHANGED);
        }
    }
}